server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1134,6 +1153,30 @@ public void run() {
     clientService = sa.server;
     log.info("Started Manager client service at {}", sa.address);
 
+    int restPort = 8999;
+    try {
+      restClientService = new EmbeddedRpcWebServer(this, restPort);
+      restClientService.addServlet(getRestServlet(), "/rest/*");
+      restClientService.start();
+
+      if (!restClientService.isRunning()) {
+        throw new RuntimeException("Unable to start rpc http server on port: " + restPort);
+      }
+    } catch (Exception e) {
+      throw new IllegalStateException("Unable to start embedded web server " + getHostname(), e);
+    }
+
+    String advertiseHost = getHostname();
+    if (advertiseHost.equals("0.0.0.0")) {
+      try {
+        advertiseHost = InetAddress.getLocalHost().getHostName();
+      } catch (UnknownHostException e) {
+        log.error("Unable to get hostname", e);
+      }
+    }
+    HostAndPort restHostAndPort = HostAndPort.fromParts(advertiseHost, restPort);
```

Comment:
I think you can use the value of `sa.address` from the code above this. In the code above the Thrift server was started on an interface, I assume we want to use the same one, just a different port.